fix: Deal with new querystrings in issue details

untitaker opened the pull request:
Fix ISSUE-296

billyvg commented on src/sentry/static/sentry/app/components/events/interfaces/utils.jsx:
```diff
@@ -56,18 +56,37 @@ export function getCurlCommand(data) {
-  result += '"';
-  return result;
+  let queryObj = {};
+  for (let [k, v] of query) {
```
why do we not stringify `query` directly?

untitaker commented:
Sorry, I forgot to link the JIRA issue. I'll try to make a brief summary here because the JIRA ticket contains too much PII

The gist is that we fixed https://github.com/getsentry/sentry/pull/11606, but a similar bug happened somewhere else: A querystring like `foo=bar&baz=bam` would previously be sent as string through the API. We changed this to send a list of lists, but didnt consider testing the UI.

Right now the clickable URL is broken in all events that have a querystring. It serializes out the above querystring as `foo,bar,baz,bam`, which is nonsense.
